.env

```
@@ -150,3 +150,7 @@ LANGSMITH_TRACING=false
 LANGSMITH_ENDPOINT="https://api.smith.langchain.com"
 LANGSMITH_API_KEY=""
 LANGCHAIN_PROJECT=""
+
+# posthog integration
+NEXT_PUBLIC_POSTHOG_KEY=""
+NEXT_PUBLIC_POSTHOG_HOST=https://eu.i.posthog.com
```

```suggestion
NEXT_PUBLIC_POSTHOG_HOST=""
```
I would leave this blank as well, user can use a different endpoint

---

Done. changed the host to blank